### PR TITLE
PYIC-4648 Add start page

### PR DESCRIPTION
**Pull request by DanCorderIPV**
### What changed

Add start page

### Why did it change

We need a start page

### Issue tracking

- [PYIC-4648](https://govukverify.atlassian.net/browse/PYIC-4648)

[PYIC-4648]: https://govukverify.atlassian.net/browse/PYIC-4648?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/kbv/steps.js [hunk 1]
```diff
@@ -6,6 +6,11 @@ module.exports = {
     resetJourney: true,
     entryPoint: true,
     skip: true,
+    next: "answer-security-questions",
+  },
+  "/answer-security-questions": {
+    resetJourney: true,
+    entryPoint: true,
```
[the remark this answers is not in the document]
Comment (DanCorderIPV) [reply src/app/kbv/steps.js:13]: I don't think it hurts and it allowed me to go straight to the intro page while developing, but I can remove them

File: src/locales/cy/pages.yml [hunk 2]
```diff
@@ -1,5 +1,26 @@
-intro:
-  h1: Intro
+answer-security-questions:
+  title: You need to answer some security questions
```
Comment (blakeyp) [inline src/locales/cy/pages.yml:2]: Page title should be translated too

Comment (DanCorderIPV) [reply src/locales/cy/pages.yml:2]: I wasn't sure about that - there is no translation in the file: https://docs.google.com/document/d/17clnZncmGrgs2LT7A4O7e1EG7KPTSRmKCsYdTe5_lo0/edit#heading=h.t1dlal7t1ftb
Who should I ask for the translation?

File: tests/browser/pages/relying-party.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ module.exports = class PlaywrightDevPage {
     this.clientId = clientId;
 
     // Starting URL properties
-    const websiteHost = process.env.WEBSITE_HOST || "http://localhost:5050";
+    const websiteHost = process.env.WEBSITE_HOST || "http://localhost:5000";
```
[the remark this answers is not in the document]
Comment (DanCorderIPV) [reply tests/browser/pages/relying-party.js:10]: As discussed I'll change all the ports to 5000 so the debug and test experience is consistent

File: src/locales/cy/pages.yml (hunk 2 above)
Comment (blakeyp) [reply src/locales/cy/pages.yml:2]: (awaiting confirmation from UCD on this)

Comment (blakeyp) [reply src/locales/cy/pages.yml:2]: Had confirmation from UCD that the page title and heading should be the same (both translated). So we only need one entry here (suggest keeping `title` and using it for both page title and h1 heading as done in #82)

File: environment-variables.md [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
 | Name                  | Description                                                         | Default               |
 | :-------------------- | :------------------------------------------------------------------ | :-------------------- |
-| API_BASE_URL          | Base URL for the backend API server                                 | http://localhost:5055 |
+| API_BASE_URL          | Base URL for the backend API server                                 | http://localhost:5090 |
```
Comment (blakeyp) [inline environment-variables.md:9]: Could you undo this change/move it to the chore PR?

File: src/locales/en/pages.yml [hunk 5]
```diff
@@ -1,5 +1,26 @@
-intro:
-  h1: Intro
+answer-security-questions:
+  title: Answer some security questions
+  h1: Answer some security questions
```
Comment (blakeyp) [inline src/locales/en/pages.yml:3]: We'd expect these to always be the same so could we just have `title` and use it in the heading too?

File: environment-variables.md (hunk 4 above)
Comment (DanCorderIPV) [reply environment-variables.md:9]: Oops, reverted it to the wrong value

File: src/locales/en/pages.yml (hunk 5 above)
Comment (DanCorderIPV) [reply src/locales/en/pages.yml:3]: As discussed that should work, but is broken due to the frontend using i18next instead of hmpo-i18n

**Review by merlinc — APPROVED** [review]
